src/eventsub/mod.rs

```diff
@@ -938,6 +938,9 @@ pub enum Status {
     /// The client failed to reconnect to the Twitch WebSocket server within the required time after a Reconnect Message.
     #[serde(rename = "websocket_failed_to_reconnect")]
     WebsocketFailedToReconnect,
+    /// The conduit is deleted
```

Comment:
maybe should be "was deleted" ?

Reply:
Yes. We could also rename all enum members to snake_case, if I didn't miss anything.